### DISCUSSION TITLE
Welcome to Evergreen Discussions 🌲

### DISCUSSION THREAD

**Discussion by mshwery** [segmentio/evergreen#1080] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Evergreen community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪. 

  As we try out Discussions, we'll likely discontinue our [Spectrum channel](https://spectrum.chat/evergreen) in favor of centralizing the conversations, issues, and pull requests to co-exist here!

  Comment below if you'd like to introduce yourself and tell us about what you do with this community! 🙂

**Comment by cstrat** [comment]
Awesome, Spectrum didn't work very well for me anyway 👍

**Comment by kud** [comment]
Nice move. Discussions on GitHub are amazing.
